sdk/search/azure-search-documents/pom.xml

```diff
@@ -11,8 +11,8 @@
     <relativePath>../../parents/azure-client-sdk-parent</relativePath>
   </parent>
 
-  <name>Microsoft Azure Cognitive Search client for Java</name>
-  <description>This package contains client functionality for Microsoft Azure Cognitive Search</description>
+  <name>Microsoft Azure client library for Cognitive Search</name>
```

Comment:
My comment here was regarding the phrase 'Microsoft Azure.*' vs 'Azure.*". In the .NET world, the understanding is that the former refers to track 1 libraries and the latter is for the refreshed track2 libraries.
I guess that distinction matters more for .NET which has lots of track1 libraries, but I was wondering if there should be uniformity for the names of all new libraries.

Reply:
There is this guideline on naming: https://azure.github.io/azure-sdk/java_introduction.html#java-maven-name

I believe this applies to all Maven packages we release, @JonathanGiles is that correct?

Reply:
What I see above appears correct, but I'm a little confused - I feel like I'm missing some context. In Java we too have dropped the 'microsoft' from our 'com.microsoft.azure' to be just 'com.azure', but we still have text descriptions refer to Microsoft Azure.